sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -944,3 +944,82 @@ DROP TABLE `geo_country_reference`;
+('Eye_QP_RETINA_defaults', 'OSVITREOUS_pvd', 'vit: PVD', 980, 0, 0, 'VITREOUS', 'PVD', '', 0, 0, 0, 'OS');
+#EndIf
+
+#IfNotRow2D list_options list_id Eye_QP_RETINA_defaults option_id OUVITREOUS_pvd
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES
+('Eye_QP_RETINA_defaults', 'OUVITREOUS_pvd', 'vit: PVD', 990, 0, 0, 'VITREOUS', 'PVD', '', 0, 0, 0, 'OU');
+#EndIf
+
+#IfNotRow2D list_options list_id Eye_QP_RETINA_defaults option_id ODVITREOUS_vh
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES
+('Eye_QP_RETINA_defaults', 'ODVITREOUS_vh', 'vit: hemorrhage', 1000, 0, 0, 'VITREOUS', 'vitreous hemorrhage', '', 0, 0, 0, 'OD');
+#EndIf
+
+#IfNotRow2D list_options list_id Eye_QP_RETINA_defaults option_id OSVITREOUS_vh
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES
+('Eye_QP_RETINA_defaults', 'OSVITREOUS_vh', 'vit: hemorrhage', 1010, 0, 0, 'VITREOUS', 'vitreous hemorrhage', '', 0, 0, 0, 'OS');
+#EndIf
+
+#IfNotRow2D list_options list_id Eye_QP_RETINA_defaults option_id OUVITREOUS_vh
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES
+('Eye_QP_RETINA_defaults', 'OUVITREOUS_vh', 'vit: hemorrhage', 1020, 0, 0, 'VITREOUS', 'vitreous hemorrhage', '', 0, 0, 0, 'OU');
+#EndIf
+
+#IfNotRow2D list_options list_id Eye_QP_RETINA_defaults option_id ODPERIPH_0
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES
+('Eye_QP_RETINA_defaults', 'ODPERIPH_0', 'p: clear field', 1022, 0, 0, 'PERIPH', '', '', 0, 0, 1, 'OD');
+#EndIf
+
+#IfNotRow2D list_options list_id Eye_QP_RETINA_defaults option_id OSPERIPH_0
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES
+('Eye_QP_RETINA_defaults', 'OSPERIPH_0', 'p: clear field', 1024, 0, 0, 'PERIPH', '', '', 0, 0, 1, 'OS');
+#EndIf
+
+#IfNotRow2D list_options list_id Eye_QP_RETINA_defaults option_id OUPERIPH_0
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES
+('Eye_QP_RETINA_defaults', 'OUPERIPH_0', 'p: clear field', 1026, 0, 0, 'PERIPH', '', '', 0, 0, 1, 'OU');
+#EndIf
```

Comment:
Note to myself:
Will remove above 3 entries since these already exist in the database.

Reply:
maybe why we're getting the pre-install page at https://four.openemr.io/h/openemr/setup.php?site=default